app/feature/feature-home/src/main/kotlin/com/hedvig/android/feature/home/home/ui/HomeDestination.kt

```diff
@@ -461,28 +464,36 @@ private fun VeryImportantMessageCard(
         containerColor = MaterialTheme.colorScheme.warningContainer,
         contentColor = MaterialTheme.colorScheme.onWarningContainer,
       ),
-      modifier = modifier,
+      modifier = modifier.fillMaxSize(),
     ) {
-      HedvigContainedSmallButton(
-        text = if (veryImportantMessage.link != null) {
-          stringResource(R.string.important_message_read_more)
-        } else {
-          stringResource(R.string.important_message_hide)
-        },
-        onClick = {
-          if (veryImportantMessage.link != null) {
-            openUrl(veryImportantMessage.link)
-          } else {
-            hideImportantMessage(veryImportantMessage.id)
-          }
-        },
-        colors = ButtonDefaults.buttonColors(
-          containerColor = MaterialTheme.colorScheme.containedButtonContainer,
-          contentColor = MaterialTheme.colorScheme.onContainedButtonContainer,
-        ),
-        textStyle = MaterialTheme.typography.bodyMedium,
-        modifier = Modifier.fillMaxWidth(),
-      )
+      Row(
+        horizontalArrangement = Arrangement.Center,
+        modifier = Modifier.fillMaxSize(),
+      ) {
+        if (veryImportantMessage.link != null) {
+          HedvigContainedSmallButton(
+            text = stringResource(R.string.important_message_read_more),
+            onClick = { openUrl(veryImportantMessage.link) },
+            colors = ButtonDefaults.buttonColors(
+              containerColor = MaterialTheme.colorScheme.containedButtonContainer,
+              contentColor = MaterialTheme.colorScheme.onContainedButtonContainer,
+            ),
+            textStyle = MaterialTheme.typography.bodyMedium,
+            modifier = Modifier.weight(1f),
+          )
+          Spacer(modifier = Modifier.width(8.dp))
+        }
+        HedvigContainedSmallButton(
+          text = stringResource(R.string.important_message_hide),
+          onClick = { hideImportantMessage(veryImportantMessage.id) },
+          colors = ButtonDefaults.buttonColors(
+            containerColor = MaterialTheme.colorScheme.containedButtonContainer,
+            contentColor = MaterialTheme.colorScheme.onContainedButtonContainer,
+          ),
+          textStyle = MaterialTheme.typography.bodyMedium,
+          modifier = Modifier.weight(1f),
+        )
+      }
```

Comment:
I didn't check out the iOS impl, but is the hide button on the right for them too?
Just by instinct I would expect the action to be on the right and the hide on the left, wdyt?